README.md

```diff
@@ -84,43 +121,32 @@ start_instance(Ami, KeyPair, UserData, Type, Zone) ->
     erlcloud_ec2:run_instances(InstanceSpec, Config).
 ```
 
-For usage information, consult the source code and refer to the API reference at:
+For usage information, consult the source code and https://hexdocs.pm/erlcloud.
+For detailed API description refer to the AWS references at:
 
 - http://docs.aws.amazon.com/AWSEC2/latest/APIReference/Welcome.html
 - http://docs.aws.amazon.com/AmazonS3/latest/API/Welcome.html
-- http://docs.amazonwebservices.com/AmazonSimpleDB/latest/DeveloperGuide/
+and other services https://aws.amazon.com/documentation/
 
 ### Using Temporary Security Credentials
 
-The access to AWS resource might be managed through [third-party identity provider](http://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-idp.html). The access is managed using [temporary security credentials](http://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_use-resources.html).  
+The access to AWS resource might be managed through [third-party identity provider](http://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-idp.html).
+The access is managed using [temporary security credentials](http://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_use-resources.html).
 
-You can provide your amazon credentials in environmental variables.
+You can provide your amazon credentials in environmental variables
 
 ```
 export AWS_ACCESS_KEY_ID=<Your AWS Access Key>
 export AWS_SECRET_ACCESS_KEY=<Your AWS Secret Access Key>
 export AWS_SECURITY_TOKEN=<Your AWS Security Token>
 ```
-
-
 If you did not provide your amazon credentials in the environmental variables, then you need to provide configuration read from your profile:
```

Comment:
I'll update the section to give preference to profile() and new() Config options instead of evn variable. however i feel that is still valuable functionality.